2021-02-01 21:46:35.188419: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-01 21:49:09.151726: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2021-02-01 21:49:12.324652: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-01 21:49:12.326730: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:04:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.545GHz coreCount: 68 deviceMemorySize: 10.76GiB deviceMemoryBandwidth: 573.69GiB/s
2021-02-01 21:49:12.326789: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-01 21:49:13.734166: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-02-01 21:49:14.027231: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2021-02-01 21:49:14.105028: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2021-02-01 21:49:15.590586: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2021-02-01 21:49:15.637036: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2021-02-01 21:49:17.105028: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-02-01 21:49:17.105350: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-01 21:49:17.109191: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-01 21:49:17.111975: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
Using TensorFlow version:  2.3.0 , GPU: 1
Using Keras version:  2.4.0
2021-02-01 21:49:17.515454: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2500115000 Hz
2021-02-01 21:49:17.518441: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x4570720 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-02-01 21:49:17.518485: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2021-02-01 21:49:18.158234: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-01 21:49:18.159279: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x45dcb90 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2021-02-01 21:49:18.159311: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): GeForce RTX 2080 Ti, Compute Capability 7.5
2021-02-01 21:49:18.180699: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-01 21:49:18.181610: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:04:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.545GHz coreCount: 68 deviceMemorySize: 10.76GiB deviceMemoryBandwidth: 573.69GiB/s
2021-02-01 21:49:18.181656: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-01 21:49:18.181688: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-02-01 21:49:18.181726: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2021-02-01 21:49:18.181750: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2021-02-01 21:49:18.181777: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2021-02-01 21:49:18.181805: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2021-02-01 21:49:18.181831: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-02-01 21:49:18.181906: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-01 21:49:18.182791: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-01 21:49:18.183613: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2021-02-01 21:49:18.202571: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-01 21:49:30.770969: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-02-01 21:49:30.771108: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2021-02-01 21:49:30.771137: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2021-02-01 21:49:30.876846: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-01 21:49:30.878782: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-01 21:49:30.880611: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10068 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:04:00.0, compute capability: 7.5)
The inputs are normalized to have a unit Gaussian distribution
Model: "CNN-POD"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
high-res-input (InputLayer)  [(None, 1, 192, 192)]     0         
_________________________________________________________________
predic_01 (Conv2D)           (None, 128, 192, 192)     3328      
_________________________________________________________________
predic_02 (BatchNormalizatio (None, 128, 192, 192)     512       
_________________________________________________________________
predic_03 (MaxPooling2D)     (None, 128, 96, 96)       0         
_________________________________________________________________
predic_04 (Conv2D)           (None, 256, 96, 96)       295168    
_________________________________________________________________
predic_05 (BatchNormalizatio (None, 256, 96, 96)       1024      
_________________________________________________________________
predic_06 (MaxPooling2D)     (None, 256, 48, 48)       0         
_________________________________________________________________
predic_07 (Conv2D)           (None, 256, 48, 48)       590080    
_________________________________________________________________
predic_08 (BatchNormalizatio (None, 256, 48, 48)       1024      
_________________________________________________________________
predic_09 (MaxPooling2D)     (None, 256, 24, 24)       0         
_________________________________________________________________
predic_10 (Conv2D)           (None, 512, 24, 24)       1180160   
_________________________________________________________________
predic_11 (BatchNormalizatio (None, 512, 24, 24)       2048      
_________________________________________________________________
predic_12 (MaxPooling2D)     (None, 512, 12, 12)       0         
_________________________________________________________________
predic_13 (Conv2D)           (None, 512, 12, 12)       2359808   
_________________________________________________________________
predic_14 (BatchNormalizatio (None, 512, 12, 12)       2048      
_________________________________________________________________
predic_15 (Conv2D)           (None, 64, 12, 12)        294976    
=================================================================
Total params: 4,730,176
Trainable params: 4,726,848
Non-trainable params: 3,328
_________________________________________________________________
None
2021-02-01 21:49:49.715712: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 1829 of 4200
2021-02-01 21:49:59.693837: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 3794 of 4200
2021-02-01 21:50:01.566445: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:221] Shuffle buffer filled.
2021-02-01 21:50:04.424204: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-02-01 21:50:21.114443: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-02-01 21:54:34.908655: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 1986 of 4200
2021-02-01 21:54:44.940757: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 4097 of 4200
2021-02-01 21:54:45.386566: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:221] Shuffle buffer filled.
Epoch 0001/0030, loss: 6.688088417053223, val_loss: 1.7544853687286377, elapsed time from start: 359.3617651462555
Epoch 0002/0030, loss: 0.596804678440094, val_loss: 0.15982235968112946, elapsed time from start: 632.4923260211945
Epoch 0003/0030, loss: 0.06382636725902557, val_loss: 0.026925895363092422, elapsed time from start: 908.258260011673
Epoch 0004/0030, loss: 0.01742994785308838, val_loss: 0.013658368960022926, elapsed time from start: 1186.2912559509277
Epoch 0005/0030, loss: 0.012886802665889263, val_loss: 0.012419143691658974, elapsed time from start: 1462.5866210460663
Epoch 0006/0030, loss: 0.01231082808226347, val_loss: 0.01210844237357378, elapsed time from start: 1740.1217904090881
Epoch 0007/0030, loss: 0.012089564464986324, val_loss: 0.011936760507524014, elapsed time from start: 2017.5865762233734
Epoch 0008/0030, loss: 0.011950087733566761, val_loss: 0.011820649728178978, elapsed time from start: 2297.0029697418213
Epoch 0009/0030, loss: 0.011843588203191757, val_loss: 0.01172661129385233, elapsed time from start: 2575.577248096466
Epoch 0010/0030, loss: 0.011752437800168991, val_loss: 0.011646464467048645, elapsed time from start: 2854.9826231002808
Epoch 0011/0030, loss: 0.011679211631417274, val_loss: 0.011579861864447594, elapsed time from start: 3134.4813992977142
Epoch 0012/0030, loss: 0.011561726219952106, val_loss: 0.011472409591078758, elapsed time from start: 3413.899034023285
Epoch 0013/0030, loss: 0.011458060704171658, val_loss: 0.011366749182343483, elapsed time from start: 3693.882803916931
Epoch 0014/0030, loss: 0.011372649110853672, val_loss: 0.011283254250884056, elapsed time from start: 3973.6820499897003
Epoch 0015/0030, loss: 0.011292173527181149, val_loss: 0.011211211793124676, elapsed time from start: 4253.329821109772
Epoch 0016/0030, loss: 0.011222903616726398, val_loss: 0.01113771740347147, elapsed time from start: 4531.920028448105
Epoch 0017/0030, loss: 0.011159502901136875, val_loss: 0.01107910368591547, elapsed time from start: 4809.735275745392
Epoch 0018/0030, loss: 0.011103878729045391, val_loss: 0.011026639491319656, elapsed time from start: 5089.013397693634
Epoch 0019/0030, loss: 0.01105170976370573, val_loss: 0.010974743403494358, elapsed time from start: 5367.94411277771
Epoch 0020/0030, loss: 0.011005766689777374, val_loss: 0.010930377058684826, elapsed time from start: 5648.765239715576
Epoch 0021/0030, loss: 0.01096485648304224, val_loss: 0.010894132778048515, elapsed time from start: 5927.990039587021
Epoch 0022/0030, loss: 0.010926278308033943, val_loss: 0.010853888466954231, elapsed time from start: 6207.677771091461
Epoch 0023/0030, loss: 0.010891900397837162, val_loss: 0.010822142474353313, elapsed time from start: 6488.460332870483
Epoch 0024/0030, loss: 0.010862331837415695, val_loss: 0.01078787725418806, elapsed time from start: 6769.6420340538025
Epoch 0025/0030, loss: 0.010831406340003014, val_loss: 0.01075675617903471, elapsed time from start: 7051.698573350906
Epoch 0026/0030, loss: 0.010802811942994595, val_loss: 0.010732052847743034, elapsed time from start: 7333.411785364151
Epoch 0027/0030, loss: 0.010779179632663727, val_loss: 0.010710837319493294, elapsed time from start: 7614.32820725441
Epoch 0028/0030, loss: 0.01075694989413023, val_loss: 0.010685984045267105, elapsed time from start: 7869.607547998428
Epoch 0029/0030, loss: 0.0107355872169137, val_loss: 0.01066779624670744, elapsed time from start: 8125.760432958603
Epoch 0030/0030, loss: 0.010718679055571556, val_loss: 0.010646859183907509, elapsed time from start: 8380.516703128815
WARNING:tensorflow:From /home/alejandro/projects/urban-flows/.env/lib/python3.7/site-packages/tensorflow/python/training/tracking/tracking.py:111: Model.state_updates (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2021-02-02 00:09:21.871882: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /home/alejandro/projects/urban-flows/.env/lib/python3.7/site-packages/tensorflow/python/training/tracking/tracking.py:111: Layer.updates (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
